### PR TITLE
docs: re-structure installation

### PR DESCRIPTION
**Pull request by hieu-jan**
The purpose of this pull request is to restructure the installation documentation (#632)

### CODE REVIEW

File: docs/docs/getting-started/install/macOS.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+1. To get the latest version of the Jan desktop app, visit the [download page for Jan](https://jan.ai/).
+2. Download the latest version of Jan for macOS by clicking on the **Download for macOS** ensure to select the **Mac Apple Chip** version if you are using a Mac with an Apple silicon processor.
+3. In your computer's `Download` folder, double-click the Jan application file. Then, drag ```Jan.app``` to the **Application** folder, making it available in the macOS Launchpad.
+4. Open Jan by clicking on the Jan icon in the Launchpad. 
+5. Add Jan to your Dock by right-clicking on the Jan icon in the Launchpad and selecting **Options** > **Keep in Dock**.
+6. Once installation is complete, you can launch Jan by clicking on the Jan icon.
+
+## Common Questions
+### Does Jan run on Apple silicon machines?
```
Comment (dan-jan) [inline docs/docs/getting-started/install/macOS.md:18]: "Silicon"

File: docs/docs/getting-started/install/macOS.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+## Installation
+
+1. To get the latest version of the Jan desktop app, visit the [download page for Jan](https://jan.ai/).
+2. Download the latest version of Jan for macOS by clicking on the **Download for macOS** ensure to select the **Mac Apple Chip** version if you are using a Mac with an Apple silicon processor.
+3. In your computer's `Download` folder, double-click the Jan application file. Then, drag ```Jan.app``` to the **Application** folder, making it available in the macOS Launchpad.
+4. Open Jan by clicking on the Jan icon in the Launchpad. 
+5. Add Jan to your Dock by right-clicking on the Jan icon in the Launchpad and selecting **Options** > **Keep in Dock**.
+6. Once installation is complete, you can launch Jan by clicking on the Jan icon.
```
Comment (dan-jan) [inline docs/docs/getting-started/install/macOS.md:15]: Personally, I don't think we need to provide this sort of step-by-step instruction. Documentation should provide the right directions for people to find things, and trust people's ability to figure things out. 

In other words, don't say "click this", "click that" unless absolutely necessary OR referring to a confusing system designed by someone else
- If you have to give that kind of instruction, then it's a design problem (should be intuitive)
- You should then raise it up to @0xSage @imtuyethan to find a design solution 

## Download Directions

I would say Jan can be downloaded from: 

1. https://jan.ai (homepage)
2. Github Releases page (give instructions on which architecture to choose)
    - Intel Macs (i.e. choose Mac x86 dmg)
    - Apple Silicon M1, M2, M3 Macs (i.e. Mac ARM dmg)

## Which Package should I download?
For Mac installation, there's a lot of issues where people don't even know there is a difference between Intel Macs and Apple Silicon Macs. Documentation should cover this difference, and explain how to find out what architecture/processor type they are on

- Processor type (i.e. "About This Mac")
- RAM

After that, we need to explain how M1, M2, M3 -> "Choose ARM". And "Intel" = "Choose x86". 
You may need to link up with @hiento09 to verify package names, as I believe he is defining it in CI/CD. 

## RAM
Local Inference is very RAM-heavy. We should have some recommendations here for hardware minimum requirements. 

I would recommend linking to external Apple documentation for "About This Mac" (vs. writing our own).

File: docs/docs/getting-started/install/overview.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- [Linux](/getting-started/install/linux)
+- [macOS](/getting-started/install/macos)
+- [Windows](/getting-started/install/windows)
+
+Jan is lightweight and can run on a variety of hardware and platform versions. You can find the specific requirements for your platform below.
+
+### Requirements for Jan
+
+#### Hardware
+Jan is a small download (<100 MB) and has a disk footprint of under 300 MB. Being a lightweight platform, it should run smoothly on modern hardware. Additionally, as you will download, store and run open-source LLM models via Jan, we recommend that you have
```
Comment (dan-jan) [inline docs/docs/getting-started/install/overview.md:19]: Are we sure we are less than <100mb? I believe we went over recently

File: docs/docs/getting-started/install/overview.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- At least 5 GB of free disk space
+- At least 8 GB of RAM
```
Comment (dan-jan) [inline docs/docs/getting-started/install/overview.md:21]: We need to explain the different requirements for RAM and GPU VRAM. 

- RAM: using traditional computer
- VRAM: using GPU

We should also talk about how RAM and VRAM sizes relate to models that you can run, e.g: 

- 8gb of RAM of VRAM can only realistically run smaller models, e.g. 3B models or quantized 7B models
- 16gb is recommended as the "minimum usable models" are mostly at 7b (e.g. Mistral 7b)
- The really good models (e.g. 70b models) require at least X gb of RAM

You may need to do some reading up on r/localllama, and have a call with @hahuyhoang411 to get up to speed on this.

File: docs/docs/getting-started/install/overview.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+#### Platform
+Jan is supported on the following platforms:
+- Linux
+- macOS (Mac Intel Chip and Apple Silicon Chip)
+- Windows (64-bit)
```
Comment (dan-jan) [inline docs/docs/getting-started/install/overview.md:27]: We should also talk about CPU and GPU support. 

CPU: 
- ARM
- x86

GPU: 
- Nvidia (thru llama.cpp)
- Talk about additional support coming soon for AMD and Intel

For the GPU portion, you will need to get 15min of @tikikun's time to grok which GPU architectures we currently support, and what we don't.

File: docs/docs/getting-started/install/windows.md [hunk 6]
```diff
@@ -2,76 +2,13 @@
...
-2. Click on the "Download for Windows" button to download the Jan Installer.
+1. To get the latest version of the Jan desktop app, visit the [download page for Jan](https://jan.ai/).
+2. Download the latest version of Jan for Windows by clicking on the **Download for Windows**
+3. Once Jan installer (jan-win-x64-{version}.exe) is downloaded, run the installer. The installation process is expected to take around a minute.
+4. By default, Jan is installed under `C:\Users\{username}\AppData\Local\Programs\Jan`
+5. Once the installation is complete, you can launch Jan by clicking on the Jan icon.
 
-![Jan Installer](../img/jan-download.png)
-
-## Step 2: Proceed the Windows Defender
```
Comment (dan-jan) [inline docs/docs/getting-started/install/windows.md:15]: These may be good to retain in the FAQ or Troubleshooting for Windows.

File: docs/docs/getting-started/install/macOS.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+## Common Questions
+### Does Jan run on Apple silicon machines? 
+Yes, Jan supports macOS Arm64 builds that can run on Macs with the Apple silicon chipsets. You can install this version of Jan by clicking on the **Download for Mac (M1/M2)** version from the [download page for Jan](https://jan.ai/).
```
Comment (dan-jan) [inline docs/docs/getting-started/install/macOS.md:19]: I would recommend an "Uninstall" section for Mac, that covers the directories to wipe clean in 
- `Application Support`

I think @0xSage covered some of them here (for Mac)
https://github.com/janhq/jan#troubleshooting

File: docs/sidebars.js [hunk 8]
```diff
@@ -43,10 +43,10 @@ const sidebars = {
           collapsible: true,
           collapsed: true,
           items: [
-            {
-              type: "autogenerated",
-              dirName: "getting-started/install",
-            },
+            "getting-started/install/overview",
```
Comment (dan-jan) [inline docs/sidebars.js:46]: I wonder whether it makes more sense to have shorter paths, i.e. `jan.ai/install/windows`

Docusaurus "sidebar hierarchy" does not require actual file and path locations to be hierarchical, i.e. you can have a top-level folder in docs called "install"

**Review by dan-jan — CHANGES_REQUESTED** [review]

File: docs/docs/getting-started/install/linux.md [hunk 9]
```diff
@@ -2,76 +2,15 @@
...
-:::
-
-```bash
-apt install nvidia-cuda-toolkit
-```
-
-Check the installation by
-
+1. To get the latest version of the Jan desktop app, visit the [download page for Jan](https://jan.ai/).
+2. Download the latest version of Jan for macOS by clicking on the **Download for Linux**.
```
[the remark this answers is not in the document]
Comment (hieu-jan) [reply docs/docs/getting-started/install/linux.md:10]: Actually, I referred to some installation docs of mass products, they mostly used `macOS`, just I decided to change from `mac` from old version to `macOS`.

File: docs/docs/getting-started/install/overview.md (hunk 3 above)
Comment (hieu-jan) [reply docs/docs/getting-started/install/overview.md:19]: Double-check it, modified to <200mb as mac version recently went over 100 mb.

File: docs/sidebars.js (hunk 8 above)
Comment (hieu-jan) [reply docs/sidebars.js:46]: Totally agree about it. I was thinking about it too. We can eliminate `getting-started` folder by moving the `install` folder under the docs

File: docs/docs/getting-started/install/linux.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (hieu-jan) [reply docs/docs/getting-started/install/linux.md]: yeah it's a typo mistake, fixed it!! Thank you!

**Comment by dan-jan** [comment]
Wait, why did we close this?

**Pull request closed without merging**